robot/Cumulus/tests/browser/engagement_plans/create_engagement_plan.robot

```diff
@@ -2,13 +2,50 @@
 
 Resource        robot/Cumulus/resources/NPSP.robot
 Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
 ...             robot/Cumulus/resources/EngagementPlanTemplatesPageObject.py
-Suite Setup     Open Test Browser
-Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+
+*** Variables ***
+${task1}  Task_1
+${task2}  Task_2
+${sub_task1}  subtask1
+
+&{engagement_fields}  Name=AutomationTestPlan
+&{fields}  Name=testplan1    task1=Task_1    task2=Task_2
 
 *** Test Cases ***
+Create Engagement Plan Template And verify save
+    [Documentation]                      Create an engagement plan from the ui. Navigate to engagementplans listings view
+    ...                                  View the details of the engagement plan created.
+
+    [tags]                                          W-038641                 feature:Engagements
+
+     Go To Page                                       Listing                            Engagement_Plan_Template__c
+     go to engagement plan page                       create
```

Comment:
I think we should navigate through clicking on new button for atleast one test case so that we are covering that button functionality

Reply:
I added one scenario to use the new button  and made the required changes